homework4/src/main/java/jackson/HarURLEntriesParser.java

```diff
@@ -0,0 +1,29 @@
+package jackson;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+
+class JacksonUsage {
+
+	public static void main(String[] args) throws JsonParseException, JsonMappingException, IOException  {
+		FileInputStream fis = new FileInputStream("src/main/resources/drivers.json");

```

Comment:
that's the correct way to locate a file. that's relative.
talking about a comment from homework 3: 
https://github.com/anastasiia-piaskovska/lits-at-031-homeworks/pull/4/files#r557714573